### PR TITLE
feat(no-release): drop support of 3.8 and add support of 3.13

### PR DESCRIPTION
**Pull request by huxuan**
Reference: https://devguide.python.org/versions/

----
📚 Documentation preview 📚: https://ss-python--758.org.readthedocs.build/en/758/

### CODE REVIEW

File: template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja
Comment (msclock) [inline template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja:1]: Lines 49-51 seems some leftover 3.8 codes.

Comment (msclock) [reply template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja:1]: Same 3.8 codes exist all ci template files and project.toml.jinja.

**Review by msclock — COMMENTED** [review]
Python 3.8 codes still exists in the template folder.

File: template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja
Comment (huxuan) [reply template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja:1]: Good catch! Seems VSCode has cheated me for code search - -

**Review by msclock — APPROVED** [review]

**Pull request merged**